The configure check fails for cross builds.
i128 is supported since rust-1.26, so it is safe for us to assume it is always present

--- a/third_party/rust/num-traits/build.rs	2020-02-14 23:57:25.985355610 +0100
+++ b/third_party/rust/num-traits/build.rs	2020-02-14 23:58:36.255710188 +0100
@@ -1,14 +1,5 @@
-extern crate autocfg;
-
-use std::env;
-
 fn main() {
-    let ac = autocfg::new();
-    if ac.probe_type("i128") {
-        println!("cargo:rustc-cfg=has_i128");
-    } else if env::var_os("CARGO_FEATURE_I128").is_some() {
-        panic!("i128 support was not detected!");
-    }
+    println!("cargo:rustc-cfg=has_i128");
 
     autocfg::rerun_path("build.rs");
 }
